docs/README.md

```diff
@@ -0,0 +1,8 @@
+# Possible requirements
+apt-get install python3-venv
+
+# Deploying to Read the Docs
+RtD builds and deploys new versions of the docs whenever you push a new commit to GitHub (though you can tag it with a specific release!)
```

Comment:
Does this release tagging happen automatically? Like If we cut a release via the Github Releases page will it Just Work(TM)?

Reply:
yeah, it's pretty powerful.
![image](https://user-images.githubusercontent.com/69007229/89815897-6cc8ce80-dafa-11ea-9d46-e4b2749d95ef.png)
if you want to add a new 'officially documented version' you'd have to add it to this list

so, for example, `latest` is my default-build version; and if I wanted to look at the stable-build version, I could look at (note the URL) https://wimax-grapl.readthedocs.io/en/stable/